app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+
+  def self.update_post_counter(author)
+    author.update(postsCounter: author.posts.size)
+  end
```

Comment:
- [ ] It would be nice if we use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_post_counter method every time a new post has been saved to database, then also make the update_post_counter method private

Reply:
Thank you for your suggestion. I implemented this.